libs/common-services/src/test/java/uk/gov/di/ipv/core/library/domain/EvidenceRequestTest.java

```diff
@@ -0,0 +1,96 @@
+package uk.gov.di.ipv.core.library.domain;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class EvidenceRequestTest {
+    @SuppressWarnings("java:S125") // Commented out code
+    // {"scoringPolicy":"gpg45","strengthScore":2}
```

Comment:
Do you know why other comments don't need to be suppressed? Would this work?
```suggestion
    // origin: {"scoringPolicy":"gpg45","strengthScore":2}
```

Reply:
SonarCloud thinks that this is commented out code rather than a real comment so it complains
I tried `Data: {"scoringPolicy":"gpg45","strengthScore":2}` SonarCloud still didn't like it :(